### PR TITLE
Add DeleteNamespace API

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
Add DeleteNamespace API.

**Why?**
Expose DeleteNamespace feature.

**How did you test it?**
Invoke API manually but will add intergration tests later.

**Potential risks**
No risks.

**Is hotfix candidate?**
No.

### CODE REVIEW

**Comment by alexshtin** [comment]
API changes is here: https://github.com/temporalio/api/pull/166

File: service/frontend/operator_handler.go [hunk 1]
```diff
@@ -276,6 +282,61 @@ func (h *OperatorHandlerImpl) ListSearchAttributes(ctx context.Context, request
...
+	if request.GetNamespace() == "" {
+		return nil, h.error(errNamespaceNotSet, scope, endpointName)
+	}
+
+	// Execute workflow.
+	wfParams := deletenamespace.DeleteNamespaceWorkflowParams{
+		Namespace: namespace.Name(request.GetNamespace()),
+		DeleteExecutionsConfig: deleteexecutions.DeleteExecutionsConfig{
+			DeleteActivityRPS:                    h.config.DeleteNamespaceDeleteActivityRPS(),
+			ConcurrentDeleteExecutionsActivities: h.config.DeleteNamespaceConcurrentDeleteExecutionsActivities(),
```
Comment (alexshtin) [inline service/frontend/operator_handler.go:307]: Workflow has 4 parameters. I decided to expose these two through dynamic config and other 2 just leave workflow defaults.

File: common/dynamicconfig/constants.go [hunk 2]
```diff
@@ -217,6 +217,13 @@ const (
 	// of Timeout and if no activity is seen even after that the connection is closed.
 	KeepAliveTimeout = "frontend.keepAliveTimeout"
 
+	// DeleteNamespaceDeleteActivityRPS is RPS per every parallel delete executions activity.
+	// Total RPS is equal to DeleteNamespaceDeleteActivityRPS * DeleteNamespaceConcurrentDeleteExecutionsActivities.
+	DeleteNamespaceDeleteActivityRPS = "frontend.deleteNamespaceDeleteActivityRPS"
```
Comment (yiminc) [inline common/dynamicconfig/constants.go:222]: why is this frontend dynamic config?

File: service/frontend/operator_handler.go (hunk 1 above)
Comment (yiminc) [reply service/frontend/operator_handler.go:307]: why use dynamic config for this? It would be much easier to just pass in as input.

**Review by yiminc — APPROVED** [review]

**Pull request merged**